src/classes/GS_AdminSetup.cls

```diff
@@ -0,0 +1,297 @@
+    All rights reserved.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2020
+* @group GS
+* @description Class to hold functionality to dispaly Checklist in GS UI.
+*/
+public with sharing class GS_AdminSetup {
+
+    private static final String DISABLED = 'Disabled';
+    /**
+     * @description Returns a list of the checklist section with each subsection.
+     * Each section is defined in GS_Admin_Checklist_Section__mdt and each subsection is defined in GS_Admin_Checklist_Item__mtd
+     * Imagine a dev needs to add an element on the checklist. He is going to go the component and
+     * finds this method is called from the FE. As this is the method which is returning the list, 
+     * it would help a lot saying where the list came from.
+     * section to display this information in UI
+     * @return List<ChecklistSection>
+     */
+    @AuraEnabled(cacheable=true)
+    public static List<ChecklistSection> getChecklist() {
+        List<GS_Admin_Checklist_Section__mdt> checklistSections = getSectionsAndSubsections();
+        List<ChecklistSection> checklistWrapers = new List<ChecklistSection>();
+        for (GS_Admin_Checklist_Section__mdt checklistSection : checklistSections) {
+            checklistWrapers.add(wrapChecklistSection(checklistSection));
+        }
+        return checklistWrapers;
+    }
+
+    /**
+     * @description This method return a list of GS_Admin_Checklist_Section__mdt and his Items 
```

Comment:
In general, dont add "This method" to the description of the apexdoc. We all already know it is a method